src/resources/locales/en.json

```diff
@@ -39,8 +39,10 @@
         "description": "Description",
         "emptyTagList": "You have no tagged ebook.",
         "entry": {
+            "continueAudiobooks": "Hör wieder auf",
             "continueReading": "Resume reading",
-            "lastAdditions": "Recently added"
+            "lastAdditions": "Recently added",
+            "lastAudiobooks": "letzte Hörbücher"
```

Comment:
My suggested English translations:
```
            "continueReading": "Resume reading",
            "continueReadingAudioBooks": "Resume listening",
            "lastAdditions": "Recently added",
            "lastAdditionsAudioBooks": "Recently added audiobooks",
```

I would also suggest a similar approach for French, German, and Spanish